src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -345,12 +345,12 @@
                                                     <DataTrigger Binding="{Binding Path=vmHilighter.State, Mode=OneWay, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=local:MainWindow}}" Value="Off">
                                                         <Setter Property="GlyphName" Value="UnSetColor"/>
                                                     </DataTrigger>
-                                                    <Trigger Property="IsMouseOver" Value="True">
+                                                    <DataTrigger Binding="{Binding RelativeSource={RelativeSource AncestorType=Button,AncestorLevel=1}, Path=IsMouseOver}" Value="True">
```

Comment:
@jalkire, I'd love to leverage the `hoverAwareFabricIconOnButtonParent` style in the controls that change their glyphs, but I'm only able to set one style. The highlighter control is used all over the place, so I'm hesitant to try to refactor it. I'd welcome any suggestions for a clean way to approach this :)

Reply:
Spoke offline with Dave. Don't know of a low-cost way to reuse the style, unfortunately ☹